Annex60/Fluid/Movers/FlowControlled_dp.mo

```diff
@@ -25,6 +25,7 @@ model FlowControlled_dp
     "Nominal pressure raise, used to normalized the filter if filteredSpeed=true"
     annotation(Dialog(group="Nominal condition"));
 
+  // fixme: shouldn't the heads not also come from the performance record?
```

Comment:
This still needs to be sorted out. I would not involve the performance records in this parameter.